research/README.md

```diff
@@ -60,6 +60,8 @@ request.
 -   [rebar](rebar): low-variance, unbiased gradient estimates for discrete
     latent variable models.
 -   [resnet](resnet): deep and wide residual networks.
+-   [sentiment_analysis](sentiment_analysis): a simple model to classify
+    document's sentiment.
```

Comment:
nit: a document's

Reply:
ack, I will add myself to the codeowners as well.